Styles/product.css

```diff
@@ -0,0 +1,273 @@
+ul {
+    margin: 0;
+}
+
+li {
+    list-style: none;
+}
+
+button {
+    background-color: transparent;
+    border: none;
+    padding: 0;
+}
+
+/* left side filter section */
+
+:root {
+    /* rating stars css variables */
+    --rating-filled-icon-color: #000;
+    --rating-filled-icon-size: 1rem;
+    --rating-filled-icon-hover-color: none;
+    --rating-filled-gap-between: 1rem;
+}
+
+.main-container {
+    /* flex properties */
+    display: flex;
+}
+
+.side-drawer {
+    width: max-content;
+    margin: 3.5em 2em;
+    margin-right: 0rem;
+}
+
+.filter-container {
+    width: 16rem;
+
+    /* flex properties */
+    display: flex;
+    flex-direction: column;
+}
+
+.header-partition {
+    border: 0.5px solid #000;
+    border-radius: 0.3em;
+    padding: 1em;
+
+    /* flex properties */
+    display: inline-flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.list-item {
+    /* flex properties */
+    display: flex;
+    gap: 1em;
+}
+
+.heading {
+    text-transform: uppercase;
+    margin: 0;
+}
+
+.clear-button {
+    cursor: pointer;
+    background-color: transparent;
+    border: none;
+    text-decoration: underline;
+    text-transform: uppercase;
+    font-weight: 500;
+}
+
+.filter-partition {
+    border: 0.5px solid #000;
```

Comment:
you can use variable for color

Reply:
actually i have asked Vikram in doubt thread regarding this issue already. Will update as per his response.